src/WTG.Analyzers/Analyzers/Visibility/VisibilityAnalyzer.cs

```diff
@@ -47,6 +47,7 @@ static void Analyze(SyntaxNodeAnalysisContext context, FileDetailCache cache)
 
 					case SyntaxKind.ProtectedKeyword:
 					case SyntaxKind.PublicKeyword:
+					case SyntaxKind.PartialKeyword:
```

Comment:
We only want to ignore partial methods, I suspect this means we will also ignore partial classes. Should probably add samples to show that the following still correctly report the diagnostic.

```csharp
private partial class Foo
{
}

internal partial class Bar
{
}
```